lib/broadway.ex

```diff
@@ -950,7 +950,9 @@ defmodule Broadway do
     metadata = Map.new(Keyword.get(opts, :metadata, []))
 
     ref = make_ref()
-    ack = {Broadway.CallerAcknowledger, {self(), ref}, :ok}
+
+    default_ack = {Broadway.CallerAcknowledger, {self(), ref}, :ok}
+    ack = Keyword.get(opts, :ack, default_ack)
```

Comment:
Probably a minor nitpick but what would you say to naming the key something a little more descriptive like `ack_module` or `acknowledger` since this will not be exposed to the user. Just seeing a key like `ack` I could see interpreting that a couple of different ways.

Reply:
:+1: for acknowledger.

Reply:
I did change it because in this way I was pulling the whole tuple from opts which was not a good idea.